microsoft_atp_safe_links/komand_microsoft_atp_safe_links/actions/decode/action.py

```diff
@@ -17,7 +17,9 @@ def run(self, params={}):
         url = params.get(Input.URL)
         try:
             results = decode_url(url)
-            return {Output.RESULT: results}
+            if results:
+                return {Output.RESULT: results}
+            return {Output.RESULT: url}
```

Comment:
I'm not sure returning the original URL is the right solution - how will they know if the decode succeeded?

Reply:
so if the results are an empty string right now we will just return the empty string and say it decoded when it didnt

Reply:
We might want to add a variable called `decoded` that's a boolean to indicate to the user.

Reply:
wouldnt the user then have to do a {{#if [decoded]}} to use the output of this plugin or the original url?

Reply:
If we had the boolean, then they could do an automated decision for `{{decoded}} = true` which is cleaner IMO

Reply:
reviving this, my thought is that most customers just want to use this in a loop to send to virus total or something. If they have to add an automated decision and have two seperate VT steps its going to make it even harder than it already is to extract the loop data.

Reply:
Thanks for bringing this back up.